### PR TITLE
Create chrome-browser-woes

### PR DESCRIPTION
**Pull request by skyhirider**
### **User description**
Creating this pull request based on feedback from https://github.com/SeleniumHQ/selenium/issues/14342

Feel free to make suggestions to the blog post.

___

### **PR Type**
documentation

___

### **Description**
- Added a new blog post titled "Two Chrome browser woes that might affect you".
- Discussed the issue of Chrome asking for search engine selection every time it is launched during automation.
- Provided a solution to bypass the search engine selection screen using `--disable-search-engine-choice-screen`.
- Discussed the issue of Chrome downloading language plugins and wasting bandwidth.
- Provided a solution to disable language plugin downloads using `--disable-features=OptimizationGuideModelDownloading,OptimizationHintsFetching,OptimizationTargetPrediction,OptimizationHints`.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>chrome-browser-woes</strong><dd><code>Added blog post on Chrome browser issues and solutions</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

website_and_docs/content/blog/2024/chrome-browser-woes

<li>Added a new blog post discussing two issues with Chrome browser.<br> <li> Provided solutions to bypass the search engine selection screen.<br> <li> Provided solutions to prevent Chrome from downloading language <br>plugins.<br>

</details>

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1853/files#diff-e139f505ec2731bae364e20540753a495a441223b7a93243f4089a6882024cd7">+25/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

### CODE REVIEW

File: website_and_docs/content/blog/2024/chrome-browser-woes [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#Two Chrome browser woes that might affect you
```
Comment (harsha509) [inline website_and_docs/content/blog/2024/chrome-browser-woes]:
```suggestion
title: 
linkTitle:
date: 
tags: ["chrome"]
categories: ["general"]
author: <name> <link to social profile handle>
description: 
---

#Two Chrome browser woes that might affect you
```

**Review by harsha509 — CHANGES_REQUESTED** [review]
HI @skyhirider,

Thanks for the informative blog! 

However, it seems a bit incomplete and is missing titles and tags. Could you please update it with these details?

I’ve provided a sample below for reference, but feel free to rewrite it in your own style.
https://gist.github.com/harsha509/7850feaceeeead02af0ed47a8579dbc1

Thanks,
Sri

**Comment by skyhirider** [comment]
@harsha509 thanks for the suggestion, updated the article. Let me know if more changes are needed.

File: website_and_docs/content/blog/2024/chrome-browser-woes [hunk 2]
```diff
@@ -0,0 +1,34 @@
+title: "Two Chrome features you should disable"
+linkTitle: "Two Chrome features you should disable"
+date: 2024-08-13
+tags: ["chrome"]
+categories: ["general"]
+author: Marek Andreansky [@skyhirider](https://www.linkedin.com/in/marekandreansky/)
+description: This blog post lists two problematic Chrome features that can affect your automation as well as a quick way to disable them.
+---
+
+## Search engine selections creen
```
Comment (harsha509) [inline website_and_docs/content/blog/2024/chrome-browser-woes]:
```suggestion
## Search engine selection screen
```

File: website_and_docs/content/blog/2024/chrome-browser-woes [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+But for automation, it does so every single time.
+
+To bypass this, use the argument below when specifying the browser options.
+
+```
+--disable-search-engine-choice-screen
+```
+
+## Chrome wastes over 60MB of you bandwidth downloading language plugins
```
Comment (harsha509) [inline website_and_docs/content/blog/2024/chrome-browser-woes]:
```suggestion
## Chrome wastes over 60MB of your bandwidth downloading language plugins
```

**Review by harsha509 — DISMISSED** [review]

**Comment by skyhirider** [comment]
Thanks for pointing out the typos, missed them when re-reading. Fixed.

**Review by diemol — APPROVED** [review]
Thank you, @skyhirider!

**Pull request merged**